.gitignore

```diff
@@ -34,6 +34,7 @@ test_results
 /doc/ui/portfolio
 
 /vendor/cmep
+/vendor/bundle
```

Comment:
It looks like the build failed when you pushed this

On Wed, Sep 16, 2015 at 12:40 PM, Bo Sun <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/nasa/earthdata-search/pull/15#discussion_r39653470>:
>
> > @@ -34,6 +34,7 @@ test_results
> >  /doc/ui/portfolio
> >
> >  /vendor/cmep
> > +/vendor/bundle
>
> Not sure. But the 'error' (in quotes, since it is not really an error -
> the task exited with 0 status) was gone after I ignored the files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nasa/earthdata-search/pull/15/files#r39653470>.
>